components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -461,6 +467,64 @@ export function CreateWorkspacePage() {
     );
 }
 
+function LocalSSHWarning(props: { className: string }) {
+    const isLocalSSHProxyEnabled = useFeatureFlag("gitpod_desktop_use_local_ssh_proxy");
+    const [isFeedbackFormVisible, setFeedbackFormVisible] = useState<boolean>(false);
+
+    if (!isLocalSSHProxyEnabled) {
+        return null;
+    }
+
+    return (
+        <div className={props.className}>
```

Comment:
**question**: Do we want to show this alert everytime the user selects VS Code Desktop, regardless if the SSH configuration has already been modified?

Reply:
Yes, since there are some people who started this experimental and they can start VS Code Desktop from browser one (which will lead ssh config update too)